### PR TITLE
Add fourth line

### PR DESCRIPTION
**Pull request by sertmer**
HI! 

I added a little dialog. The cat now greets the sailor with "SUP DAWG".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,5 @@
 
 Today, a sailor went on a walk and was happy to see the sunshine.
 The sailor felt woozy walking on land because it doesn't rock like the sea.
-Along came a kitty cat who jumped on the sailor's shoulders.
\ No newline at end of file
+Along came a kitty cat who jumped on the sailor's shoulders.
+The cat whispered in the sailor's ear: 'SUP DAWG'.
```
Comment (cbdallavalle) [inline README.md]: I love that the cat can talk! I think it should also ask do you have any fish?

**Review by cbdallavalle — CHANGES_REQUESTED** [review]
Let's change the dialogue of the cat first!

**Comment by sertmer** [comment]
Great Idea! I added that line! Let me know if there is anything else!

**Review by cbdallavalle — APPROVED** [review]
Looks good!

**Pull request merged**